ui/onboarding/KeysMain.qml

```diff
@@ -3,88 +3,88 @@ import QtQuick.Controls 1.3
 import QtQuick.Controls 2.3
 import QtQuick.Layouts 1.3
 import "../shared"
+import "../imports"
 
 Page {
+    id: page
     property alias btnExistingKey: btnExistingKey
     property alias btnGenKey: btnGenKey
+    
+    Item {
+        id: container
+        width: 425
+        height: {
+            let h = 0
+            const children = this.children
+            Object.keys(children).forEach(function (key) {
+                const child = children[key]
+                h += child.height + Theme.padding
+            })
+            return h
```

Comment:
One trick here:

If we expect a single child instead (e.g. an `Item`) that groups the other ones, we can simply do `height: children[0].height + Theme.padding`

That `Item` needs a `height` as well, which it would determine by its own children etc. Not super familiar with this part of the code, but if the number of children is unknown at compile time, then that pattern works really well.

Reply:
Sadly, if I did that trick, I would end up with the same need of the loop, because the new wrapper (eg the `Item`) would still not have the right height unless I set a hardcoded height or do the loop just like here.